.github/workflows/s3-deployment.yml

```diff
@@ -0,0 +1,24 @@
+name: AWS deploy
```

Comment:
You're doing continuous deplyment.  Nice!

Reply:
Yes, that area is my strength :)